.github/workflows/arena-brazil-testing-deploy.yml

```diff
@@ -6,7 +6,7 @@ on:
         type: choice
         description: What release to deploy
         options:
-          - game_backend
+          - main_server
```

Comment:
These options will be removed in #449

Reply:
Pointed this PR to that one so that we wouldn't get conflicts!